src/uct/base/uct_iov.inl

```diff
@@ -27,7 +27,7 @@
 static UCS_F_ALWAYS_INLINE
 size_t uct_iov_get_length(const uct_iov_t *iov)
 {
-    return iov->count * iov->length;
+    return iov->length;
```

Comment:
is it possible to remove `uct_iov_get_length()` function at all? or it is needed for UCS IOV infra?

Reply:
There is a related method for iovec
```cpp
size_t ucs_iovec_get_length(const struct iovec *iov)
{
    return iov->iov_len;
}
```
E.g. uct_iov_get_length and ucs_iovec_get_length are passed to ucs_iov_total_length macro to calculate total length of array of uct_iov_t and iovec arrays respectively.

Reply:
yes, then ok to have it